src/__tests__/ReduxRouter-test.js

```diff
@@ -223,8 +223,6 @@ describe('<ReduxRouter>', () => {
       'or a class/function (for composite components) but got: object. ' +
       'Check the render method of `ReduxRouterContext`.';
 
-    const routingContextInvalidPropErrorMessage = 'Invalid prop `RoutingContext` of type `object` supplied to `ReduxRouterContext`';
```

Comment:
No `npm ls | grep react` shows me `react@0.14.8`. Seems to be the correct minor version.